src/django/api/management/commands/sync_databases.py

```
@@ -0,0 +1,861 @@
+            new_record = model_class(**record_data)
+            new_record.save()
+        except Exception as e:
+            logger.error(f'Error inserting record in {model_name}: {e}')
+            raise
+
+    def _update_record(self, source_record, target_record,
+                       foreign_keys, model_name, excluded_fields, pk_type):
+        '''Update an existing record in the target database.'''
+        if self.__dry_run:
+            logger.info(f'[DRY RUN] Would update record with sync_field '
+                        f'{getattr(source_record, "uuid")} in {model_name}')
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)